agi.go

```diff
@@ -408,11 +434,37 @@ func (a *AGI) StreamFile(name string, escapeDigits string, offset int) (digit st
 
 // Verbose logs the given message to the verbose message system
 func (a *AGI) Verbose(msg string, level int) error {
-	return a.Command("VERBOSE", msg, strconv.Itoa(level)).Err()
+	return a.Command("VERBOSE", strconv.Quote(msg), strconv.Itoa(level)).Err()
+}
+
+// Verbosef logs the formatted verbose output
+func (a *AGI) Verbosef(format string, args ...interface{}) error {
+	return a.Verbose(fmt.Sprintf(format, args...), 9)
 }
 
 // WaitForDigit waits for a DTMF digit and returns what is received
 func (a *AGI) WaitForDigit(timeout time.Duration) (digit string, err error) {
 	resp := a.Command("WAIT FOR DIGIT", toMSec(timeout))
-	return string(resp.Result), resp.Error
+	resp.ResultString = ""
+	if resp.Error == nil && resp.Result >= 32 {
```

Comment:
I would prefer to see this using [`strconv.IsPrint`](https://golang.org/pkg/strconv/#IsPrint), for formalism.

Reply:
Agree.